src/backend/joanie/core/templates/issuers/invoice.html

```diff
@@ -0,0 +1,149 @@
+{% load i18n %}
+{% load static %}
+{% load extra_tags %}
+
+<html>
+  {% if debug %}
+  <head>
+    <style>
+      {{ css }}
+    </style>
+  </head>
+  {% endif %}
+  <body class="{%if debug %}debug {% endif %}invoice">
+    <header class="invoice-header">
+      <div class="header-logo">
+        <img src="{% base64_static "joanie/images/logo_fun.png" %}" />
+        <img src="{{ organization.logo }}" />
+      </div>
```

Comment:
Copy pasta fail ? We do not provide organization logo through the document context.

Reply:
Can you just try to generate a PDF document to ensure all is rendered properly ?

Reply:
[example_invoice.pdf](https://github.com/openfun/joanie/files/13819781/example_invoice.pdf)
 :top: After some corrections, here is the result

Reply:
After some corrections, here is the result of the invoice template : 
[example_invoice.pdf](https://github.com/openfun/joanie/files/13819822/example_invoice.pdf)